addon/utils/load-relationship.js

```diff
@@ -7,6 +7,7 @@ export default function loadAll(model, relationship, dest, options = {}) {
         page: page
     };
     query = Ember.merge(query, options || {});
+    Ember.set(model, 'query-params', query);
```

Comment:
Is this a special field on the model that we control? Do we need to clear the field in between requests?

The two scenarios I'm considering here are:
- The same model gets interrogated later and possibly remembers a previous setting, thus not starting fresh, and
-  In some cases, setting extra fields on a model to store information can cause the model to be marked as dirty, even if the fields have no real impact on a request